lib/staticizer/crawler.rb

```diff
@@ -164,6 +164,16 @@ def save_page_to_aws(response, uri)
       # Upload this file directly to AWS::S3
       opts = {:acl => "public-read"}
       opts[:content_type] = response['content-type'] rescue "text/html"
+
+      # Detect a meta-redirect and set an S3 hosting redirect metadata item
+      if response =~ /META http-equiv='refresh' content='0;URL="(.*)"/
```

Comment:
I think there are likely some common cases where this regex would fail. For example:

`<meta http-equiv="refresh" content="0;url=http://example.com" />`

or

`<meta http-equiv="refresh" content="2;url='http://example.com'" />`

I'll merge this request and then likely modify this to catch a wider range of meta redirects.